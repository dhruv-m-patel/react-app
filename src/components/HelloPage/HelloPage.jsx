import React from 'react';
import { Link } from 'react-router-dom';
import Jumbotron from 'react-bootstrap/Jumbotron';
import Container from 'react-bootstrap/Container';
import Image from 'react-bootstrap/Image';
import Page from '../Page';

export default function HelloPage() {
  return (
    <Page>
      <Jumbotron fluid>
        <Container>
          <h1>Hello World!</h1>
          <Link to="/">See Welcome Screen</Link>
        </Container>
      </Jumbotron>
      <Container>
        <Image src="/images/web.png" alt="" roundedCircle style={{ maxWidth: '300px' }} />
      </Container>
    </Page>
  );
}
