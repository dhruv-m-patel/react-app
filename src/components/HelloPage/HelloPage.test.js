import Tester from '../../../tests/Tester';
import HelloPage from './HelloPage';

const tester = new Tester();

describe('HelloPage', () => {
  test('it should render', () => {
    const snapshot = tester.getSnapshot(HelloPage);
    expect(snapshot).toMatchSnapshot();
  });
});
