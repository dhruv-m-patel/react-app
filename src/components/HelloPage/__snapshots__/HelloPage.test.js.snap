// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloPage it should render 1`] = `
<div
  className="page container"
>
  <div
    className="jumbotron jumbotron-fluid"
  >
    <div
      className="container"
    >
      <h1>
        Hello World!
      </h1>
      <a
        href="/"
        onClick={[Function]}
      >
        See Welcome Screen
      </a>
    </div>
  </div>
  <div
    className="container"
  >
    <img
      alt=""
      className="rounded-circle"
      src="/images/web.png"
      style={
        Object {
          "maxWidth": "300px",
        }
      }
    />
  </div>
</div>
`;
